ci-operator/step-registry/hypershift/jira-agent/process/hypershift-jira-agent-process-commands.sh

```
@@ -321,6 +322,133 @@ send_slack_notification() {
+    --permission-mode default \
+    --verbose \
+    --output-format stream-json \
+    "$@" \
+    | grep '^{'
+  local rc=${PIPESTATUS[0]}
+  # Collect OTEL JSONL from agentic-ci temp dirs
+  for f in /tmp/agentic-ci-run.*/claude-otel.jsonl; do
+    [ -f "$f" ] && cat "$f" >> "${OTEL_LOG}"
+  done
+  rm -rf /tmp/agentic-ci-run.*
+  return $rc
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)